src/deadline/job_attachments/README.md

```diff
@@ -33,6 +33,47 @@ RootPrefix/
 
 [ja-security]: https://docs.aws.amazon.com/deadline-cloud/latest/userguide/security-best-practices.html#job-attachment-queues
 
+### S3 Key Format
+
+#### Data Files
+
+Data files in the job attachments system are stored using a content-addressable approach. The S3 key format for data files is:
+
+```
+RootPrefix/Data/<file_hash>.<hash_algorithm>
+```
+
+For example:
+```
+my-deadline-prefix/Data/a1b2c3d4e5f6g7h8i9j0.xxh128
+```
+
+Where:
+- `<file_hash>` is the hash of the file contents
+- `<hash_algorithm>` is the algorithm used to generate the hash (currently "xxh128")
+
+This content-addressable approach ensures that identical files are only stored once, regardless of their original filenames or paths.
+
+#### Manifest Files
+
+Manifests are stored for both job inputs and task outputs. The S3 key for the input manifest for a job can be found by [calling GetJob](https://docs.aws.amazon.com/deadline-cloud/latest/APIReference/API_GetJob.html#API_GetJob_ResponseSyntax) and looking in the response under `attachments.manifests.rootPath`. See the [developer guide on job attachments](https://docs.aws.amazon.com/deadline-cloud/latest/developerguide/run-jobs-job-attachments.html#job-attachments-in-depth) for more info. While the input manifest location is not formally defined, in the current job attachments implementation it follows the pattern:
```

Comment:
Instead of "formally defined," can you describe than it is set by the call to CreateJob and the pattern cannot be relied on?

Reply:
Removed the pattern since it's an internal implementation detail and will only cause confusion